File: University/AvvalBiniS1.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        try (Scanner scanner = new Scanner(System.in)) {

            int min = scanner.nextInt();
            int max = scanner.nextInt();
            boolean flag = true;
            String result = "";

            for(int i = min + 1; i < max; i++) {
                for (int j = 2; j <= i/2; j++) {
                    if (i % j == 0) {
                        flag = false;
                        break;
                    }
                }
                if (flag)
                    result = result.concat(i + ",");

                flag = true;
            }

            if (result.length() > 1)
                System.out.print(result.substring(0, result.length() - 1));

        }
    }
}

// <Ctrl+Z>

File: University/AvvalBiniS1.rb
class Main
    def self.main()
        try ()
        {
            min = gets.chomp
            max = gets.chomp
            flag = true
            result = ""
            i = min + 1
            while (i < max)
                j = 2
                while (j <= i / 2)
                    if (i % j == 0)
                        flag = false
                        break
                    end
                    j += 1
                end
                if (flag)
                    result = result.concat(i.to_s + ",")
                end
                flag = true
                i += 1
            end
            if (result.length > 1)
                print(result[0...result.length - 1])
            end
        }
    end
end
Main.main()
